src/Exception/RequestAwareTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Http\Client\Exception;
+
+use Psr\Http\Message\RequestInterface;
+
+trait RequestAwareTrait
+{
+    /**
+     * @var \Psr\Http\Message\RequestInterface
```

Comment:
annotation can also profit from the `use` statement

Reply:
Apologies, I don't think I understand... do we want a `@uses \Http\Client\Exception\RequestAwareTrait` on each of the exceptions or a `@used-by ::setRequest()` and `@used-by ::getRequest()` on this private `$request` property? (or something else?)

Reply:
Ah, this doesn't need to be a fully qualified namespace, that makes sense - will update